hack/verify-golint.sh

```diff
@@ -30,7 +30,12 @@ fi
 cd "${KUBE_ROOT}"
 
 GOLINT=${GOLINT:-"golint"}
-bad_files=$($GOLINT -min_confidence=0.9 ./...)
+
+find_files() {
+  $(go list -f '{{.Dir}}' ./... | grep -v vendor)
+}
+
+bad_files=$(find_files | xargs $GOLINT -min_confidence=0.9)
```

Comment:
This doesn't work. Multiple arguments to golint are supposed to be files, not package directories...